### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by jvegaraf**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by jvegaraf2 — COMMENTED** [review]
Sounds good

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
 <script>
   // Initialization
```
Comment (jvegaraf2) [inline index.html:53]: This is just a suggestion

Comment (jvegaraf2) [reply index.html:53]: just acomment

**Review by jvegaraf — COMMENTED** [review]
OK

**Review by jvegaraf2 — COMMENTED** [review]
Done

**Pull request merged**